### PR TITLE
reports: 20191212 Labitconf in Montevideo, Uruguay

### PR DESCRIPTION
**Pull request by 3lian**
Report Labitconf, Montevideo, Uruguay.

### CODE REVIEW

**Comment by xaur** [comment]
@3lian thanks for the report. Amazing work at Labitconf!

One question: was it 1000 or 500 visitors?

**Comment by 3lian** [comment]
Thanks! Absolutely worthwhile conference to be present. 

1000 attendees, I first thought 500 but after checking with the organisers they confirmed 1000.

File: reports/20191212-labitconf-montevideo-uruguay.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+- Andreas Antonopoulous - when asked about decentralised governance he acknowledged that governance is going to matter more over time as public blockchain grow.
+- Jimmy Song - when asked about decentralised governance he mentioned that governance was unavoidably centralised and that term was in it self an oxymoron.
+- Adam Back - we briefly talk and took a picture with Adam Back.
+- Rodolfo Andragnes organiser of Labitconf and founder of Bitcoin Argentina, we had several conversations about the next Labitconf in Guatemala and the participation of Decred next year.
+- Alejandro de la Torre VP from Poolin, we met and talk about the role of Poolin in Decred mining.
+- Diego Saldivar CEO de RIF labs, founder of RSK.
+- Pablo Coirolo from Aeternity Blockchain.
+- Rocelo Lopes from Stratum.
+- Pablo Gonzales CEO, Eduardo Arenas Listing Manager and Jose Rodriguez VP Payments from Bitso, we talk about having DCR listed next year after they settle their operations in Argentina and Brazil.
```
Comment (3lian) [inline reports/20191212-labitconf-montevideo-uruguay.md]: Pablo Gonzales CEO, Eduardo Arenas Listing Manager and Jose Rodriguez VP Payments from Bitso, we talked about new opportunities for Latin America, particularly the interest of Decred and Bitso in Argentina and Brazil.

File: reports/20191212-labitconf-montevideo-uruguay.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+- Tim Byun CEO OKCoin, Jim Nguyen VP Marketing, Oscar Muñoz Compliance Officer, Pablo Almagro Regional Manager Latam, we talk about having a cohosted event between OKCoin and Decred.
+- Rafael Bonventi Client Success Specialist, Lane Fraum Account Manager at Kraken Exchange, talked about having DCR listed and get introduced to the listing team to follow up on the conversation.
```
Comment (3lian) [inline reports/20191212-labitconf-montevideo-uruguay.md]: Rafael Bonventi Client Success Specialist, Lane Fraum Account Manager at Kraken Exchange, talked about opportunities between Kraken and DCR and get introduced to the listing team to follow up on the conversation.

**Comment by xaur** [comment]
Merged, final link:

https://github.com/decredcommunity/events/blob/master/reports/20191212-labitconf-montevideo-uruguay.md

**Pull request closed without merging**